Description: Fixes manpage-has-errors-from-man
Auhtor: Herbert Parentes Fortes Neto <hpfn@ig.com.br>
Last-Update: 2015-08-15
Index: libcdk5-5.0.20141106/man/cdk_binding.3
===================================================================
--- libcdk5-5.0.20141106.orig/man/cdk_binding.3
+++ libcdk5-5.0.20141106/man/cdk_binding.3
@@ -72,7 +72,7 @@ which is one of the following values.
 center tab(/);
 l
 l l
-lw13 lw30 lw 30 .
+lw10 lw20 lw20 .
 \fBEObjectType_Value/Corresponding_Widget/Widget_Manual_Page\fR
 =
 vALPHALIST/Alphalist Widget/cdk_alphalist (3)
@@ -152,7 +152,7 @@ Otherwise, a few special cases are perfo
 center tab(/) allbox;
 l l
 l l
-lw35 lw35 .
+lw10 lw35 .
 \fBKey/Result\fR
 =
 CTRL-A/KEY_HOME
